### ISSUE TITLE
Not finding Elements when a View is wrapped with NavigationStack

### ISSUE THREAD

**Issue by domfz** [nalexn/ViewInspector#207]
I have a view which has a NavigationStack around it.
```
var body: some View {
return NavigationStack {
            VStack {
                Text("Hello")
                ..........
            }
```
Now when trying to find the text "Hello" via (I use quick and nimble for testing, but XCTest gives the same result)
```
                expect(try homeView.inspect()
                    .find(text: "Hello")
                ).toNot(beNil())
```
it gives me a failed test. When removing the NavigationStack the test succeeds.
Also when adding a tag to the NavigationStack as `.tag` property and print the output of 
```
expect(try homeView.inspect()
                    .find(viewWithTag: "NavigationStack")
```
(this test also succeeds)

it gives me the whole hierarchy which clearly shows `(SwiftUI.Text(storage: SwiftUI.Text.Storage.anyTextStorage(<LocalizedTextStorage: 0x00006000010656d0>: "Hello")` at some point inside of it.

Am I missing something or is there a bug when traversing through Views wrapped in a NavigationStack via ViewInspector?

**Comment by ian0cordova** [comment]
Same issue here with NavigationSplitView under the new ios 16 Navigation setup

**Comment by ian0cordova** [comment]
> Same issue here with NavigationSplitView under the new ios 16 Navigation setup

Things I've tried

- Updating to ViewInspector 0.9.2 by switching from Pods to SPM
- extending NavigationSplitView and NavigationStack to conform to inspectable

Still getting 

` testView(): failed: caught error: "Search did not find a match. Possible blockers: InternalBody"`

**Comment by nalexn** [comment]
Fixed released with v0.9.4
